LBH.AdultSocialCare.Api/V1/Controllers/DayCarePackageController.cs

```diff
@@ -119,5 +134,47 @@ public async Task<ActionResult<IEnumerable<DayCarePackageResponse>>> GetDayCareP
             var result = await _updateDayCarePackageUseCase.Execute(dayCarePackageId, dayCarePackageForUpdateDomain).ConfigureAwait(false);
             return Ok(result);
         }
+
+
+
+        #region DayCarePackageOptions
+
+        /// <summary>
+        /// Gets the term time consideration option list.
+        /// </summary>
+        /// <returns>List of term time consideration options</returns>
+        /// <response code="200">Returns term time consideration option list</response>
+        [ProducesResponseType(typeof(IEnumerable<TermTimeConsiderationOptionResponse>), StatusCodes.Status200OK)]
+        [HttpGet("term-time-considerations")]
```

Comment:
I think this may justify having a DayCareTermTimeConsiderationsController

Reply:
> I think this may justify having a DayCareTermTimeConsiderationsController

This returns options we require under day care package. Since data for this is seeded and doesn't look like we'll need create, update and delete on this, do we need a controller with a single action only?